src/main/kotlin/agallipo/BoardingCardReader.kt

```diff
@@ -0,0 +1,5 @@
+package agallipo
+
+interface BoardingCardsReader {
+    fun readCards() : MutableList<String>
```

Comment:
I've seen you tend to use Mutable objects. Mutable objects can usually make things easier because you can just work with the same object and modify it. But can you think on problems they can bring? ;) 
Also, there is a matter of semantic here: in this case, your interface is a "reader". What I understand with that is that its only mission is to read things. So, Why returning a mutable list if semantically speaking it cannot modify it?

Reply:
Yep you're right, I started doing it that way because I got a mess with the syntax of Kotlin and was easier for me to treat all the variables as the same type, I've now change the boardingCards as a List, because that will never change.